jfix-armeria-facade/src/test/kotlin/ru/fix/armeria/facade/it/JFixTestWebfluxServerApi.kt

```diff
@@ -22,9 +23,9 @@ interface JFixTestWebfluxServerApi {
         const val DELAYED_PARTS_PATH = "$BASE_PATH/delayedParts"
     }
 
-    @GET(DELAYED_ANSWER_PATH)
+    @GET("$DELAYED_ANSWER_PATH/{delayMs}")
     suspend fun delayedAnswer(
-        @Query("delayMs") delayMs: Long,
+        @Path("delayMs") delayMs: Long,
```

Comment:
since we don't use query param anymore we have no param name and have possible misunderstand what measurement units should be used in path. For exmaple, "/somepath/30" - what it meant to be? 30 seconds (30000 millis) or 30 milliseconds? Will it be documented anyhow?

Reply:
> delay**Ms**

I think Ms ending makes param's semantics quite obvious, isn't it?

the problem I solved by converting query param to path param is to include param's value into metric label. It is required to distinguish metrics of different request types (big delay and small one)